### DISCUSSION TITLE
请问怎么修改公式行间距？

### DISCUSSION THREAD

**Discussion by Burning1997** [sjtug/SJTUThesis#1005] [提问]
![微信图片_20240111155006](https://github.com/sjtug/SJTUThesis/assets/58418855/2a0cd13a-a57a-4b3c-a962-8e599259e91d)
![微信图片_20240111155207](https://github.com/sjtug/SJTUThesis/assets/58418855/810e50bc-7e77-460f-940d-c491661009e7)

评审老师说公式2-4到2-8之间的行间距太大，请问怎么修改？

**Comment by LogCreative** [comment]
改用 `align` 环境。

```latex
\begin{align}
a=b+c\\
d=e+f
\end{align}
```
